sunpy/spectra/tests/test_spectrogram.py

```diff
@@ -14,6 +14,16 @@
     Spectrogram, LinearTimeSpectrogram, _LinearView
 )
 
+from sunpy.spectra.spectrum import Spectrum
+
+def test_spectrum():
+    #The idea is to create two 1-D arrays, data and freq_axis,
+    #Pass them to the Spectrum object and check for equality of dimensions accordingly.
+    data = np.linspace(0, 100, 100)
+    freq = np.linspace(1, 100, 100)
+    spec = Spectrum(data,freq)
+    if not (np.shape(spec.data) == np.shape(spec.freq_axis)):
+        raise ValueError
```

Comment:
Alas ! It is confusing, I was told on IRC, to create a test function (after reading the testing docs) in test_spectrogram.py. 
P.S. I have included the error check in spectrum.py file since that is where the spectrum object is declared. It works too, declaring data and freq_axis arrays of different dimensions , does raise the requisite error and works fine when their dimensions are equal.